### DISCUSSION TITLE
Security Hardened Sandbox and app installed on non system partition

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#2664] [Q&A Feedback] [answered]
Hello,

What else do I need to add to sandboxie.ini to allow an app from non-system partition to run in hardened sandbox?

```
ReadFilePath=<StartRunAccess>,X:\app_path\*
ProcessGroup=<StartRunAccess>,app.exe,<what_else?>
```

It does not matter what app is that. Adding just app.exe is not enough to make it run due to

```
---------------------------
Sandboxie Start
---------------------------
Could not invoke program:



"X:\a\app.exe"

System Error Code:

Access is denied.

 (5)
---------------------------
OK   
---------------------------
```

With `ProcessGroup=<StartRunAccess>,*` the app works

**Comment by DavidXanatos** [comment]
try just NormalFilePath=X:\app_path\*

**Comment by offhub** [comment, marked as answer]
<what_else?> -> `Start.exe`

**Reply by ghost** [reply]
Right, thanks. It is not needed when an app we want to run is installed on systemdrive and necessary otherwise
